src/server/generic_family.cc

```diff
@@ -351,8 +351,8 @@ void Renamer::Finalize(Transaction* t, bool skip_exist_dest) {
 
 OpStatus Renamer::MoveSrc(Transaction* t, EngineShard* es) {
   if (es->shard_id() == src_sid_) {  // Handle source key.
-    // TODO: to call PreUpdate/PostUpdate.
-    auto it = es->db_slice().FindExt(t->GetDbContext(), src_res_.key).first;
+    auto res = es->db_slice().FindMutable(t->GetDbContext(), src_res_.key);
+    auto it = res.it;
```

Comment:
auto&

Reply:
Copying iterators is trivial and we do it all the time (it has 2 pointers and 2 bytes), is there any particular reason to use a ref here?

Reply:
Anyway, done :)